sktime/utils/validation/forecasting.py

```diff
@@ -25,9 +25,18 @@
 from sklearn.base import clone, is_regressor
 from sklearn.ensemble import GradientBoostingRegressor
 
-from sktime.utils.validation import is_date_offset, is_int, is_timedelta
+from sktime.utils.validation import (
```

Comment:
kindly add yourself as an author to files where you make substantial changes

Reply:
added